src/main/java/com/stacksurveyor/backend/controllers/SurveyController.java

```diff
@@ -31,45 +31,43 @@
 
 @CrossOrigin
 @RestController
+@RequestMapping("/surveys")
 public class SurveyController {
     @Autowired
     SurveyRepository surveyRepository;
 
-
-    @GetMapping("/survey/@me")
+    @GetMapping("/@me")
     public ResponseEntity<List<Survey>> getAllSurveysOfUser() {
         // TODO Change impl when JWT Authentication is completed
         UUID userId = new UUID(2, 2);
 
-        return new ResponseEntity<>(surveyRepository.findAllByUserId(userId), HttpStatus.ACCEPTED);
+        return new ResponseEntity<>(surveyRepository.findAllByUserId(userId), HttpStatus.OK);
     }
 
-    @PostMapping("/survey")
-    public ResponseEntity<Survey> createSurvey(@RequestBody SurveyAddForm surveyAddForm) {
+    @PostMapping("/create")
+    public ResponseEntity<Survey> createSurvey(@RequestBody SurveyCreateForm surveyCreateFormForm) {
         // TODO Change impl when JWT Authentication is completed
         UUID userId = new UUID(2, 2);
 
-        Survey survey = new Survey(Uuids.timeBased(), userId,
-                surveyAddForm.getTitle(), surveyAddForm.getDescription());
-
-        surveyRepository.save(survey);
+        Survey survey = surveyRepository.save(new Survey(Uuids.timeBased(), userId,
+                surveyCreateFormForm.getTitle(), surveyCreateFormForm.getDescription()));
 
         return new ResponseEntity<>(
                 survey,
                 HttpStatus.CREATED
         );
     }
 
-    @GetMapping(value = "/survey/{id}")
-    public ResponseEntity<Survey> getSurvey(@PathVariable("id") UUID id) throws SurveyException {
-        Survey survey = surveyRepository.findSurveyById(id);
+    @GetMapping(value = "/{id}")
+    public ResponseEntity<Survey> getSurvey(@PathVariable("id") String id) throws SurveyException {
```

Comment:
also, can i know why you changed it to a String instead of UUID here ?

Reply:
Because if you put it as a UUID and the server tries to directly convert an invalid string to UUID, then it will show internal server error instead of "Survey not found". That's why I take the UUID in the request as a String, and then convert it to UUID. Then if there's any issue, it just throws 'Survey not found'

Reply:
oh ok ; didnt think of that